test/integration/integration_suite_test.go

```diff
@@ -52,7 +52,6 @@ var (
 	k8sClients    map[string]client.Client
```

Comment:
This map is not required, since there be only one cluster.

Reply:
fixed